person.rb

```diff
@@ -29,6 +30,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def rent_book(rental)
+    @rentals << rental
+  end
 end
 
 class BaseDecorator < Nameable
```

Comment:
- [ OPTIONAL] kindly I can suggest, instead of creating the class for BaseDecorator in person.rb file the best practice is to create a new file for the class basedecorator it will make your code looks clear and more readable :+1: